packages/webapp/src/util/animal.ts

```diff
@@ -38,3 +38,8 @@ export const generateInventoryId = (animalOrBatch: Animal | AnimalBatch): string
   const key = 'count' in animalOrBatch ? AnimalOrBatchKeys.BATCH : AnimalOrBatchKeys.ANIMAL;
   return `${key}_${animalOrBatch.id}`;
 };
+
+export const parseInventoryId = (inventoryId: string): { kind: string; id: number } => {
```

Comment:
Can `AnimalOrBatchKeys` be used instead of `string`?

Reply:
I did have to type cast what's returned from the split; I don't know if that's unavoidable or not.

Reply:
I tried myself and did the same too 😅